priv/repo/migrations/20180410174012_modify_session_id_type_to_message_log.exs

```diff
@@ -0,0 +1,30 @@
+defmodule Aida.Repo.Migrations.ModifySessionIdTypeToMessageLog do
+  use Ecto.Migration
+  alias Aida.Repo
+
+  def up do
+    Repo.query! "UPDATE message_logs AS m SET session_id = session_uuid"
```

Comment:
Shouldn't this be done _after_ the alter column?

Reply:
The problem is that the data of the column must have the right type when the 'ALTER COLUMN TYPE' is executed. Otherwise it raises an error. So the proper values are set before perorming the ALTER COLUM